### PR TITLE
APPS-361 - updated findProject org parameter

### PR DESCRIPTION
**Pull request by mhrvol**

### CODE REVIEW

File: src/python/dxpy/cli/exec_io.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def interactive_help(in_class, param_desc, prompt):
             if opt_num == 0:
                 query_project = dxpy.WORKSPACE_ID
             elif opt_num == 1:
-                query_project = dxpy.find_one_project(name="Reference Genome Files", public=True, billed_to="org-dnanexus", level="VIEW")['id']
+                query_project = dxpy.find_one_project(name="Reference Genome Files", public=True, billed_to="org-dnanexus_apps", level="VIEW")['id']
```
Comment (commandlinegirl) [inline src/python/dxpy/cli/exec_io.py:142]: Looks good,

Please update the prompt (line 133) --> to "Reference Genome Files"

**Review by commandlinegirl — CHANGES_REQUESTED** [review]

File: src/python/dxpy/cli/exec_io.py (hunk 1 above)
Comment (mhrvol) [reply src/python/dxpy/cli/exec_io.py:142]: fixed

**Review by commandlinegirl — APPROVED** [review]
LGTM

**Comment by commandlinegirl** [comment]
Note, a part of this was done in another PR https://github.com/dnanexus/dx-toolkit/pull/667 by @emiloslavsky (please merge main)

**Pull request merged**